liquidcore/site/urls.py

```diff
@@ -19,6 +20,9 @@
     path('accounts/profile', views.profile, name='profile'),
     path('o/', include('oauth2_provider.urls', namespace='oauth2_provider')),
     path('system-health', views.healthchecks_page, name='healthchecks'),
+    path('.well-known/openid-configuration/',
```

Comment:
comment where this confusion comes from (BBB integration) - otherwise we risk having lots of random rules and not know why they're in here

Reply:
comment added